src/Microsoft.ApplicationInsights.AspNetCore/project.json

```diff
@@ -76,18 +76,18 @@
         //    "version": "1.2.0-beta2",
         //    "type": "build"
         //},
-        "System.Threading.Tasks.Analyzers": {
-            "version": "1.2.0-beta2",
-            "type": "build"
-        }
-        //"Text.Analyzers": {
-        //    "version": "1.2.0-beta2",
-        //    "type": "build"
-        //},
-        //"XmlDocumentationComments.Analyzers": {
-        //    "version": "1.2.0-beta2",
-        //    "type": "build"
-        //},
+      "System.Threading.Tasks.Analyzers": {
+        "version": "1.2.0-beta2",
+        "type": "build"
+      }
+      //"Text.Analyzers": {
+      //    "version": "1.2.0-beta2",
+      //    "type": "build"
+      //},
+      //"XmlDocumentationComments.Analyzers": {
+      //    "version": "1.2.0-beta2",
+      //    "type": "build"
+      //},
```

Comment:
nitpick: this formatting didn't need to change, especially not to become inconsistent.